server/src/routes/project.ts

```diff
@@ -0,0 +1,67 @@
+import { PrismaClient } from "@prisma/client";
+import express from "express";
+
+export const projectRoutes = express.Router();
+const prisma = new PrismaClient();
```

Comment:
You don't need to create a new prisma client. Instead, just import the one from `common.ts`.